topological_inventory-providers-common.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'config', '~> 1.7', '>= 1.7.2'
   spec.add_runtime_dependency "activesupport", "~> 5.2.2"
-  spec.add_runtime_dependency "manageiq-loggers", "~> 0.4.0", ">= 0.4.2"
+  spec.add_runtime_dependency "manageiq-loggers", "~> 0.5.0", ">= 0.5.0"
```

Comment:
@lpichler `>= 0.5.0` is not necessary when values are the same

Reply:
@slemrmartin updated 👍